crates/strategies/blend-strategies/src/liquidation_strategy.rs

```diff
@@ -248,8 +231,23 @@ impl BlendLiquidator {
                         user,
                         pending_fill.target_block.clone()
                     );
-                    self.pending_fill.push(pending_fill);
+                    self.pending_fill.push(pending_fill.clone());
                 }
+
+                let status = if pending_fill.pct_to_fill > 0 {
+                    "picked up"
+                } else {
+                    "ignored"
+                };
+                let msg = format!(
+                    "Liquidator: {} has {} a new user liquidation auction for user: {:?} with lot: {:?} and bid: {:?}",
```

Comment:
we probably want to leave the `!<channel>` tag here for now, I think. It's important to get a phone notification when these get picked up still.

Reply:
(this log is a significant improvement tho)